Walkthrough.md

```diff
@@ -36,7 +36,6 @@ Inside BTCPay you can create and manage an unlimited number of stores. Each stor
 
 - General settings - Configure individual store settings, set up derivation scheme for your wallet and adjust the number of confirmations or invoice expiration time. Server admins can enable Lightning Network payments.
 - Rates - Set the source for cryptocurrency to fiat rates for your store.
-- Checkout experience - [Customize the appearance](https://nbitstack.com/t/how-to-modify-btcpay-css-checkout-page/99/2) of the checkout page, choose default coin, etc.
```

Comment:
No need to remove this, here's the link in docs https://docs.btcpayserver.org/faq-and-common-issues/faq-serversettings#how-to-modify-the-checkout-page